app/models/gql/gql.rb

```diff
@@ -335,14 +335,13 @@ def inspect
     private
 
     def apply_initializer_inputs
-      set_initializer_inputs(:present)
-      set_initializer_inputs(:future)
+      %i(present future).each(&method(:set_initializer_inputs))
```

Comment:
Personally, I think this is less readable than it was before. I also recently learned that `&method(name)` isn't such a good idea since it performs quite poorly compared to the more explicit `each { |thing| name(thing) }`.

```
Calculating -------------------------------------
             &method    388.031k (± 4.2%) i/s -      1.963M in   5.067859s
        each { ... }      1.779M (± 4.2%) i/s -      8.918M in   5.021471s
```